test/gui/test_managergui.py

```diff
@@ -4,7 +4,11 @@
 import mock
 
 from test import stubs
-from subscription_manager.gui import managergui, registergui
+try:
+    from subscription_manager.gui import managergui, registergui
+except Exception as exc:
+    import unittest
+    unittest.skip(f"Cannot run tests: {exc}")
```

Comment:
all the tests here are tagged with `gui`, so maybe a cleaner way could be to add `!gui` to the `attr` of the nosetest config file